src/lib/components/resource/ResourceDetailCard.tsx

```diff
@@ -1,49 +1,103 @@
 import {
-  Accordion,
   AccordionButton,
+  AccordionIcon,
   AccordionItem,
   AccordionPanel,
-  Button,
   Flex,
   Text,
 } from "@chakra-ui/react";
 
+import { CopyButton } from "../copy";
 import { CustomIcon } from "../icon";
+import type { InternalResource } from "lib/types";
 
 interface ResourceDetailCardProps {
-  name: string;
+  resourceData: InternalResource;
 }
-export const ResourceDetailCard = ({ name }: ResourceDetailCardProps) => {
+
+export const ResourceDetailCard = ({
+  resourceData,
+}: ResourceDetailCardProps) => {
+  const moveResourceObject = JSON.parse(resourceData.moveResource);
```

Comment:
This will make an error `Uncaught SyntaxError: Unexpected end of JSON` when parsing `""`. So, I prefer to use `parseSchemaInitialData` with fallback value.